scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -314,10 +314,10 @@ object H2OMOJOCache extends H2OMOJOBaseCache[EasyPredictModelWrapper, H2OMOJOMod
     config.setConvertUnknownCategoricalLevelsToNa(model.getConvertUnknownCategoricalLevelsToNa())
     config.setConvertInvalidNumbersToNa(model.getConvertInvalidNumbersToNa())
     if (canGenerateContributions(config.getModel)) {
-      config.setEnableContributions(model.getWithDetailedPredictionCol())
+      config.setEnableContributions(model.getWithContributions() && model.getWithDetailedPredictionCol())
```

Comment:
I think this was another bug which was there -> we should enable contributions only if both contributions and detailed prediction col are true